.pipelines/singletenancy/cilium-overlay/cilium-overlay-e2e-step-template.yaml

```diff
@@ -158,6 +149,9 @@ steps:
   - script: |
       echo "validate pod IP assignment and check systemd-networkd restart"
       kubectl get pod -owide -A
+      # Deleting echo-external-node deployment until cilium version matches TODO. https://github.com/cilium/cilium-cli/issues/67 is addressing the change.
```

Comment:
what this test is for?

Reply:
It tries to echo (curl in this case) to an external, non-cilium, node. The deployment is stuck in pending due to its node selector looking for "cilium.io/no-schedule": "true". We cannot remove or skip the test at this point.

Reply:
for now its fine, we should contribute to cilium add skip check to this test case @vipul-21